common/static/common/js/discussion/views/discussion_content_view.js

```diff
@@ -363,30 +363,30 @@
             };
 
             DiscussionContentShowView.prototype.toggleEndorse = function(event) {
-                var beforeFunc, is_endorsing, msg, updates, url,
+                var isEndorsing, msg, updates, url,
                     self = this;
                 event.preventDefault();
-                is_endorsing = !this.model.get('endorsed');
+                isEndorsing = !this.model.get('endorsed');
                 url = this.model.urlFor('endorse');
                 updates = {
-                    endorsed: is_endorsing,
-                    endorsement: is_endorsing ? {
+                    endorsed: isEndorsing,
+                    endorsement: isEndorsing ? {
                         username: DiscussionUtil.getUser().get('username'),
                         user_id: DiscussionUtil.getUser().id,
                         time: new Date().toISOString()
                     } : null
                 };
                 if (this.model.get('thread').get('thread_type') === 'question') {
-                    if (is_endorsing) {
-                        msg = gettext('We had some trouble marking this response as an answer.  Please try again.');
+                    if (isEndorsing) {
+                        msg = gettext('This response could not be marked as an answer. Refresh the page and try again.');  // eslint-disable-line max-len
                     } else {
-                        msg = gettext('We had some trouble removing this response as an answer.  Please try again.');
+                        msg = gettext('This response could not be marked as not an answer. Refresh the page and try again.');  // eslint-disable-line max-len
```

Comment:
@andy-armstrong instead of "marked as not an answer" -> "...could not be unmarked as an answer"